modules/olm-disabling-copied-csvs.adoc

```diff
@@ -12,6 +12,11 @@ When Operators are configured to use the `AllNamespaces` install mode, versus ta
 
 To support these larger clusters, cluster administrators can disable copied CSVs for Operators installed with the `AllNamespaces` mode.
 
+[WARNING]
+====
+Disabling copied CSVs restricts access to the full OLM experience to cluster administrators. Only cluster administrators can interact with Operators installed with `AllNamespaces` mode. Users can only interact with Operators directly installed in their namespace. 
```

Comment:
I think some key information is missing from the existing warning:
- Copied CSVs are used to identify which operators area available in a given namespace.
- The operator hub console is effectively disabled for users.
- The operators are still available in theses namespaces and users can still rely on them. The existing wording ("interact") seems to indicate otherwise.
```suggestion
Copied CSVs identify which operators are available in a given namespace. By disabling copied CSVs:
- The functionality made available through the operatorhub page will limited by the RBAC available to users. For example, users will not be able to see operators that are installed in a separate namespace but configured to reconcile events in their namespaces.
- On cluster, users will not be aware which operators are available in their namespace unless the operator was installed directly in their namespace.
```